dusty/cli/__init__.py

```diff
@@ -96,6 +98,12 @@ def main():
     if '-d' in sys.argv:
         return run_daemon()
 
+    if '-v' in sys.argv:
+        sock = _connect_to_daemon()
+        daemon_check = Payload(version.check_version, constants.VERSION)
+        daemon_version = _run_command(sock, daemon_check.serialize())
+        return
```

Comment:
Also you're duplicating all this logic here, don't want to do that unless necessary